docs/github_projects_guidelines.md

```diff
@@ -323,9 +314,8 @@ a project and should be always kept in sync.
 
 ### Creating releases
 
-Since we are following the [GitFlow] branching model, a GitHub release is always
-created after a release branch is merged into the master branch and the tag has
-been created.
+Since we are following the [trunk-based development] branching model, a GitHub
+release is created directly from a commit that was tagged with release tag.
```

Comment:
```suggestion
Since we are following the [trunk-based development] branching model, a GitHub
release is created directly from a commit that was tagged with the release tag.
```